### PR TITLE
Move postinstallcmds changes to a seperate hook.

### PR DESCRIPTION
**Pull request by bartoldeman**
This cuts down on the noise from parse_hook, and postinstallcmds
is only relevant for the installed easyconfig, not the dependencies.

### CODE REVIEW

**Comment by bartoldeman** [comment]
tested with intel-compilers-2021.2.0.eb to make sure it gets the postinstallcmds correct.

File: cc_hooks_gentoo.py [hunk 1]
```diff
@@ -824,7 +824,9 @@ def pre_configure_hook(self, *args, **kwargs):
     orig_enable_templating = self.cfg.enable_templating
     self.cfg.enable_templating = False
 
-    modify_all_opts(self.cfg, opts_changes)
+    modify_all_opts(self.cfg, opts_changes,
+        opts_to_skip=['builddependencies', 'dependencies', 'modluafooter',
```
Comment (mboisson) [inline cc_hooks_gentoo.py:828]: why the addition of these options to skip ?

Comment (bartoldeman) [reply cc_hooks_gentoo.py:828]: because the default option list has those (see `cc_hooks_common.py`) Should we edit it there?

Comment (bartoldeman) [reply cc_hooks_gentoo.py:828]: https://github.com/ComputeCanada/easybuild-computecanada-config/blob/4077e531881e9a8030c502564effc1354c22e4df/cc_hooks_common.py#L30

Comment (mboisson) [reply cc_hooks_gentoo.py:828]: ah, I see.

**Pull request merged**